deployments/charts/kuma/README.md

```diff
@@ -9,6 +9,7 @@ The chart supports Helm v3+.
 
 | Parameter                                   | Description                                                                       | Default                              |
 |---------------------------------------------|-----------------------------------------------------------------------------------|--------------------------------------|
+| `global.namespace     `                     | The namspace where all resources will be installed                                | `kuma-system`                        |
```

Comment:
What's the reason to use this vs .Release.Namespace?

Reply:
Nikolay is on PTO this week. I'm not sure what was the motivation but I changed to `.Release.Namespace` everywhere and it seems to work fine

Reply:
`.Release.Namespace` defaults to `default` unless you specify it on the command line. I wanted to default to `kuma-system`